docs/2-Interviews/preparing_for_interviews.md

```diff
@@ -12,22 +13,57 @@ Be prepared to:
 - talk about and work towards writing an algorithm to solve a problem, during the interview
 - you can also expect a “tell me about yourself” question (so be prepared!) and some time where they tell you about their company
 
+### What material could show up in an interview?
+
+This is a difficult question to answer. The more things you know, the less likely it is that an interview question could trip you up. That being said, you have limited time to learn things. There a few concepts that are very likely to show up. There's also a long list of algorithms that _could maybe_ show up once in a while, depending on the company, the interviewer or whether venus and mercury are in the same quadrant of the solar system this week. You could be spending an arbitrarily long time preparing for all possible eventualities.
+
+You can do that if you enjoy it, but it's healthy to set a timebox. That is, allocate yourself a certain amount of time in advance to prepare for interviews. Learn as much as you can during that time, but no more. And in that time, you probably want the biggest bang for the buck.
+
+Very likely to show up in interviews:
+- Understanding O() analysis
```

Comment:
for people that don't know what this is, maybe add (this is how the time/space an algorithm uses is usually measured) or something like that